seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -87,9 +94,49 @@ def handle(self, *args, **options):
 
         logger.info('DONE')
 
+    def _report_validation_errors(self, **kwargs):
+        reported_runs = self._get_runs_for_file_name(ERRORS_REPORTED_FILE_NAME, **kwargs)
+        reported_run_ids = {run_dict['run_version'] for run_dict in reported_runs.values()}
+
+        error_runs = self._get_runs_for_file_name(VALIDATION_ERRORS_FILE_NAME, **kwargs)
+        messages = []
+        reported_run_paths = set()
+        for path, run_dict in error_runs.items():
+            if run_dict['run_version'] in reported_run_ids:
+                continue
+
+            error_summary = json.loads(next(line for line in file_iter(path)))
+            summary = [
+                f'Callset Validation Failed',
+                f'Projects: {error_summary["project_guids"]}',
+                f'Reference Genome: {run_dict["genome_version"]}',
+                f'Dataset Type: {run_dict["dataset_type"]}',
+                f'Run ID: {run_dict["run_version"]}',
+                f'Validation Errors: {error_summary["error_messages"]}',
+                f'See more at https://storage.cloud.google.com/{path}'
+            ]
+            messages.append('/n'.join(summary))
+            reported_run_paths.add(path)
+
+        if messages:
+            safe_post_to_slack(
+                SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL, '\n\n'.join(messages),
+            )
+            with TemporaryDirectory() as temp_dir_name:
```

Comment:
while it is only writing one file, I think using the `write_multiple_files` helper function in export utils would allow you to reuse most of that code and not need to reimplement so much boiler plate

Reply:
I think the usage would just be
```
write_multiple_files([(ERRORS_REPORTED_FILE_NAME, [], [])], run_directory, user)
```